docs/tutorials/helloworldintro.md

```diff
@@ -16,23 +21,27 @@ This "Hello World" guide walks you through creating a project, installing the ML
 This tutorial requires an MLAPI-supported version of Unity (2019.4+).
 :::
 
- ![Open Unity create new project](/img/openunity.gif)
```

Comment:
please delete any removed gif files from the static/img folder(s) thanks!

Reply:
All animated gifs now removed from whole site and replaced with videos